src/validations.mjs

```diff
@@ -0,0 +1,304 @@
+'use strict';
+export const validateBuyRecord = validate10;
```

Comment:
Is it strictly necessary to commit this file? Feels like it opens us up to the risk of having the same logic encoded in two different places:
1) schema file
2) generated schema file output based on the schema file
I'd worry about the second not being generated after changes to the first and stale code getting shipped.

Reply:
no - not necessary, i included it so you can see the outputs that ajv generates and injects in to the build

Reply:
generating the updated schema is part of the build script. so it's reproduced on any builds `e.g. yarn build or node build.js`